[2021-07-16 16:42:58,509] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:42:07.654941+00:00 [queued]>
[2021-07-16 16:42:58,543] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:42:07.654941+00:00 [queued]>
[2021-07-16 16:42:58,544] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:42:58,545] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 16:42:58,545] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:42:58,561] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T16:42:07.654941+00:00
[2021-07-16 16:42:58,575] {standard_task_runner.py:52} INFO - Started process 9028 to run task
[2021-07-16 16:42:58,581] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T16:42:07.654941+00:00', '--job-id', '1200', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpa77zefvt', '--error-file', '/tmp/tmpfe79geq9']
[2021-07-16 16:42:58,582] {standard_task_runner.py:77} INFO - Job 1200: Subtask restart_test
[2021-07-16 16:42:58,653] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:42:07.654941+00:00 [running]> on host 13346afafa5b
[2021-07-16 16:42:58,750] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T16:42:07.654941+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T16:42:02.902551+00:00
[2021-07-16 16:43:02,193] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T164207, start_date=20210716T164258, end_date=20210716T164302
[2021-07-16 16:43:02,258] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 16:43:02,335] {local_task_job.py:151} INFO - Task exited with return code 0
